src/components/Gallery.js

```diff
@@ -12,6 +14,14 @@ const AboutContainer = ({ author, description, license, filePage, close }) => {
     onKeyLeft: () => { window.open(filePage) }
   })
 
+  useLayoutEffect(() => {
+    const descriptionNode = document.querySelector('.description')
```

Comment:
Please don't use global selectors. Put a ref on the top div or the specific element you need.

Reply:
true, I think the above screenshot caused by this global reason. thanks for the catching.